README.md

```diff
@@ -14,30 +14,53 @@ A GitHub Action which helps enforce a minimum code coverage threshold.
 
 ## Inputs
 
-### `path`
+VeryGoodCoverage accepts the following configuration inputs:
 
-**Optional** The path to the `lcov.info` file.
+| Input name   | Description                                                                                                                                                                     | Default                | Example value                         | Optional |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------- | ------------------------------------- | -------- |
+| path         | The path to the lcov.info file.                                                                                                                                                 | './coverage/lcov.info' | 'packages/my_app/coverage/lcov.info'  | ✅       |
+| min_coverage | The minimum coverage percentage allowed.                                                                                                                                        | 100                    | 95                                    | ✅       |
+| exclude      | List of paths to exclude from the coverage report, separated by an empty space. Supports [globs](<https://en.wikipedia.org/wiki/Glob_(programming)>) to describe file patterns. | ''                     | '**/\*\_observer.dart **/change.dart' | ✅       |
 
-**Default** `./coverage/lcov.info`
-
-### `min_coverage`
+## Example usage
 
-**Optional** The minimum coverage percentage allowed.
+```yaml
+uses: VeryGoodOpenSource/very_good_coverage@v2
+with:
+  path: './coverage/lcov.info'
+  min_coverage: 95
+  exclude: '**/*_observer.dart **/change.dart'
+```
 
-**Default** 100
+## FAQs
 
-### `exclude`
+#### How can I avoid VeryGoodCoverage reporting an empty or non-existent coverage file?
 
-**Optional** List of paths to exclude from the coverage report, separated by an empty space. Supports `globs` to describe file patterns.
+[Relevant issue](https://github.com/VeryGoodOpenSource/very_good_coverage/issues/167)
 
-## Example usage
+A failure for non-existent coverage file can be resolved by setting the path input to match the location of the already generated lcov file.
 
 ```yaml
 uses: VeryGoodOpenSource/very_good_coverage@v2
 with:
-  path: './coverage/lcov.info'
-  min_coverage: 95
-  exclude: '**/*_observer.dart **/change.dart'
+  path: 'my_project/coverage/lcov.info'
+```
+
+If your generated lcov file is empty this might be because you have no test files or your tests are not generating any coverage data.
+
+If you wish to always bypass these warnings we recommend using a conditional statement in your workflow to avoid running the VeryGoodCoverage action when the lcov file is empty or non-existent.
```

Comment:
```suggestion
If you wish to always bypass these warnings, we recommend using a conditional statement in your workflow to avoid running the Very Good Coverage action when the lcov file is empty or non-existent.
```